src/index.js

```diff
@@ -147,7 +149,13 @@ export default class ScrubThumbnailsPlugin extends UICorePlugin {
           onLoaded()
         }
       }
-      $img.one("load", onImgLoaded).attr("src", thumb.url)
+      // put image in dom to prevent browser removing it from cache
+      if (cachedImageUrls.indexOf(thumb.url) === -1) {
+        // not been cached
+        cachedImageUrls.push(thumb.url)
```

Comment:
I don't think `cachedImageUrls` is actually used anywhere now so can be removed?

Previously the images were added to the dom but set to `display: none` because I thought this would make chrome keep them in memory, but I don't actually think that was working properly anyway so I'm fine with removing it.